###############################################
########## Physical robot parameters ##########
###############################################

robot:
  name: "yumi_single_arm"
  ip_robot: "192.168.125.1"
  robot_port_rws: 443
  # ip_robot: "192.168.56.101" # for virtual robot (robotstudio)
  # robot_port_rws: 80 # for virtual robot (robotstudio)
  name_robot: "ROB_1"
  task_robot: "T_ROB1"
  pos_corr_gain: 1
  max_speed_deviation: 300
  robot_port_egm: 6511
  root_name: "single_yumi_base_link"
  tip_name: "gripper_base"
  is_free_drive_on: false

  tool_data:
    # tcp_pose       : [0.0, 0.0, 125.2, 1.0, 0.0, 0.0, 0.0] # x, y, z, w, x, y, z. Positions in [mm]
    tcp_pose       : [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0] # x, y, z, w, x, y, z. Positions in [mm]
    mass           :  0.230                                  # [kg]
    cog            : [-17.3, 13.6, 49.0]                   # [mm]
    inertia_tensor : [0.0  ,  0.0,  0.0]                   # [kg*m^2]
    aom            : [1.0  ,  0.0,  0.0, 0.0] 

###############################################
#### Settings for ros_control control loop ####
###############################################

robot_hw_control_loop:
  loop_hz: 250
  cycle_time_error_threshold: 0.004

###############################################
# Settings for ros_control hardware interface #
###############################################

robot_hardware_interface:
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
    - joint_7